content/en/tracing/profiling/_index.md

```diff
@@ -0,0 +1,91 @@
+Datadog Profiling is in beta. Reach out to <a href="/help/">Datadog Support</a> if you encounter any issues or have feedback to share.
+</div>
+
+Profiling libraries are shipped within the following tracing language library. Refer to the dedicated language page to learn how to enable profile connections for your application:
+
+{{< partial name="profiling/profiling-languages.html" >}}
+
+## Profile explorer
+
+Once your profile collection is set up, profiles are available in the [APM > Profiling][1] page within Datadog:
+
+{{< img src="tracing/profiling/profiling_main_page.png" alt="Profiling main page">}}
+
+Each line represents a profile for a process on which Datadog profiling was running for a short amount of time. Depending on the language, these processes are profiled between 15s and 60s.
+
+### Search
+
+You can filter according to infrastructure tags or application tags set up from your [environment tracing configuration][2]. By default the following facets are available:
+
+|  Facet   |  Definition                                                             |
+| -------- | ----------------------------------------------------------------------- |
+| Env      | The environment your application is running on (`prod`, `staging`).     |
+| Service  | The name of the [service][3] your code is running.                      |
+| Version  | The version of your code.                                               |
+| Host     | The host name your profiled process is running on.                      |
+| Runtime  | The type of runtime the profiled process is running (`JVM`, `CPython`). |
+| Language | The language of your code (`Java`, `Python`).                           |
+
+The following measures are available:
+
+|  Measure          | Definition                                                                                                                                                                          |
+| ----------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| CPU               | CPU load summed on all cores of the process. This means this can go above `100%` and the theoretical limit is `nCores` \* `100%`.                                                    |
+| Memory Allocation | Memory allocation rate in `Bytes/s` over the course of the profile. This value can be above the maximum amount of RAM memory because it can be garbage collected during the profile. |
+
+## Profile
+
+Click on one line to view a specific profile:
+
+{{< img src="tracing/profiling/profile.png" alt="A specic profile">}}
+
+The profile header contains information associated with your profile, like: the service that generated it, or the environment and code version associated to it. In the upper right corner of each profile, there are options to:
+
+- Download the profile
+- Switch the profile to full screen
```

Comment:
These should go at the end if at all; they're the ~least important thing on the panel.

Reply:
Those are feature proposed by the profile modal and are available for all languages so they have to be documented.

Reply:
Ok, then they should be at the end.